### ISSUE TITLE
Add new dedicated page for user success stories

### ISSUE THREAD

**Issue by solvingj** [conan-io/website#282]
We are creating several user success stories with Conan users. We will soon need a dedicated page on conan.io for this content.  The main page has a user-focused section, but it is simply not suitable for the new content.  As we create this new page, we will also need to decide what to do about navigation, as there is currently not a great way to make such a page discoverable with the current conan.io navigation.

**Comment by solvingj** [comment]
We decided to have a look at how JFrog website is modeling these customer success stories, and I'm glad we did, it looks really nice... https://jfrog.com/resource-center/usecases/

It's a super-simple gallery of customers with a link to get to a dedicated page for each story. We recognized that we already have a similar gallery with the tribe (https://conan.io/tribe.html).  So, we decided we could probably use this same gallery widget for our own customer success story landing page and simply replace the pictures with the company logos, and then "view profile" would be come "read story".
